indra/newview/llviewerjointattachment.h

```diff
@@ -98,6 +98,17 @@ class LLViewerJointAttachment :
 	typedef std::vector<LLPointer<LLViewerObject> > attachedobjs_vec_t;
 	attachedobjs_vec_t mAttachedObjects;
 
+    bool    hasChanged(const LLVector3 &pos, const LLQuaternion &rot) const
+    {
+        return (!(pos - mLastTrackedPos).isNull() || !LLQuaternion::almost_equal(rot, mLastTrackedRot));
```

Comment:
A thought experiment here... Right now I'm looking for ANY change in the position or rotation. 
Suppose, instead we look at the magnitude of the change instead. For position the magnitude of the change is greater than X% or the angle between the vectors is greater than some value. (This would mean that the further away from the center the less resolution, unless magnitude was taken into account.)  For rotation the angle between the two rotations is greater than some amount. (say 5degrees) ...